protocol/identity/profile_showcase.go

```diff
@@ -122,25 +120,5 @@ func Validate(preferences *ProfileShowcasePreferences) error {
 		return ErrorNoAccountProvidedWithTokenOrCollectible
 	}
 
-	accountsMap := make(map[string]*ProfileShowcaseAccountPreference)
-	for _, account := range preferences.Accounts {
-		if _, ok := accountsMap[account.Address]; ok {
-			return ErrorDublicateAccountAddress
-		}
-		accountsMap[account.Address] = account
-	}
-
-	for _, collectible := range preferences.Collectibles {
-		account, ok := accountsMap[collectible.AccountAddress]
-		if !ok {
-			return nil
-			// NOTE: with current wallet collectible implementation we don't know account on this stage
-			// return errorNoAccountAddressForCollectible
-		}
-		if account.ShowcaseVisibility < collectible.ShowcaseVisibility {
-			return ErrorAccountVisibilityLowerThanCollectible
-		}
-	}
-
```

Comment:
I think it's still makes sense to keep this here.
Here we check that the `ProfileShowcasePreferences` has a valid configuration. And then we validate the collectibles ownership.

Reply:
i'm going to delete `AccountAddress` from the collectible entry because we don't have such field in the source model